src/NexusMods.Abstractions.Games/RunGameTool.cs

```diff
@@ -127,9 +127,10 @@ public virtual async Task Execute(Loadout.ReadOnly loadout, CancellationToken ca
         
     }
 
-    private async Task<CommandResult> RunCommand(CancellationToken cancellationToken, AbsolutePath program)
+    private async Task<CommandResult> RunCommand(CancellationToken cancellationToken, string[] arguments, AbsolutePath program)
```

Comment:
We should also update `RunWithShell()` to accept command line parameters, similar to `RunCommand()`